drs_downloader/manager.py

```diff
@@ -9,14 +9,27 @@
 import os
 import tqdm
 import tqdm.asyncio
+import sys
 
 
 from drs_downloader import DEFAULT_MAX_SIMULTANEOUS_OBJECT_RETRIEVERS, DEFAULT_MAX_SIMULTANEOUS_PART_HANDLERS, \
     DEFAULT_MAX_SIMULTANEOUS_DOWNLOADERS, DEFAULT_MAX_SIMULTANEOUS_OBJECT_SIGNERS, DEFAULT_PART_SIZE, MB, GB
 
 from drs_downloader.models import DrsClient, DrsObject
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger()
+
+stdout_handler = logging.StreamHandler(sys.stdout)
+stdout_handler.setLevel(logging.DEBUG)
+
+file_handler = logging.FileHandler('logs.log')
```

Comment:
Please use a more descriptive name for the log file, such as _<program_name>_`.log`
For example: `drs_downloader.log`

Reply:
[Fixed](https://github.com/anvilproject/drs_downloader/blob/feature/manager/drs_downloader/manager.py#L26)

Reply:
[Fixed](https://github.com/anvilproject/drs_downloader/blob/feature/manager/drs_downloader/cli.py#L99-L100) and [tested](https://github.com/anvilproject/drs_downloader/blob/feature/manager/tests/integration/test_cli_options.py#L19) ("Downloading to...")